lib/cartopy/examples/scalar_data/eyja_volcano.py

```diff
@@ -7,8 +6,9 @@
 demand from the Stamen tile server. Internally these tiles are then combined
 into a single image and displayed in the cartopy GeoAxes.
 
+.. _examples-eyja_volcano:
 """
-__tags__ = ["Scalar data"]
+# -*- coding: utf-8 -*-
```

Comment:
ah you are right - I'm trying to remember how we fixed this in MPL...maybe we just removed those lines? Or it was fine w/o moving them?